ruby/vampires.rb

```diff
@@ -0,0 +1,56 @@
+puts "How many employees you will be processed?"
+num_empl = gets.chomp.to_i 
```

Comment:
This is a good direction with variable naming though take it a step further and eliminate abreviations; `employees_to_process` , `employee_total` , etc.